docs/development/setup-advanced.md

```diff
@@ -136,6 +137,116 @@ Store.
    environment](../development/setup-vagrant.html#step-4-developing),
    ignoring the parts about `vagrant` (since you're not using it).
 
+## Using the Vagrant Hyper-V provider
+
+You should have [Vagrant](https://www.vagrantup.com/downloads) and
+[Hyper-V](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v)
+installed on your system. Ensure they both work as expected.
+
+**NOTE**: Hyper-V is available only on Windows Enterprise, Pro, or Education.
+
+1. Get started by cloning the repository to your local machine.
+
+   ```bash
+   git clone --config pull.rebase git@github.com:YOURUSERNAME/zulip.git
+   ```
```

Comment:
The version of this in the main instructions has one more step, adding zulip/zulip as remote `upstream`:
https://zulip.readthedocs.io/en/latest/development/setup-vagrant.html#step-2-get-zulip-code

This should include that, and probably also include the same links that that bit of instructions has.